src/engine/UIManager.hxx

```diff
@@ -30,6 +18,7 @@ struct GameMenu
 };
 
 struct BuildMenu;
+using json = nlohmann::json;
```

Comment:
Why did you add this? The only other changes you've made to this file are adding a field, changing function declaration headers, and renaming things from LayoutGroup to Layout. Nowhere in this do you have any JSON parsing.

Reply:
because you marked json in some files, not in one place, and it used later in this file, look down

Reply:
Where in the file? Can you please give a line number or paste the code?

Reply:
![image](https://user-images.githubusercontent.com/918081/180622968-07d21201-6c93-40f5-9f01-59ed097ad460.png)